### PR TITLE
Add new tests for product types

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I added tests for product types

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://stable.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5678 Delete product type and remove attribute](https://app.clickup.com/t/2549495/SALEOR-5678)

File: cypress/support/api/requests/Product.js [hunk 1]
```diff
@@ -113,12 +113,16 @@ export function createProduct({
     attributeValue,
     `values:["${attributeValue}"]`
   );
+  const attributes = getValueWithDefault(
+    attributeId,
+    `attributes:[{
+    id:"${attributeId}"
+    ${attributeValuesLine}
+  }]`
```
Comment (Cloud11PL) [inline cypress/support/api/requests/Product.js:121]: I think formatting is a bit off

File: cypress/support/api/requests/ProductType.js [hunk 2]
```diff
@@ -143,3 +143,22 @@ export function setProductTypeAsDigital(productTypeId, isDigital = true) {
...
   return cy.sendRequestWithQuery(mutation);
 }
+
+export function assignAttribute(
+  productTypeId,
+  attributeId,
+  attributeType = "VARIANT"
+) {
+  const mutation = `mutation{
+    productAttributeAssign(productTypeId:"${productTypeId}" operations:{
```
Comment (Cloud11PL) [inline cypress/support/api/requests/ProductType.js:153]:
```suggestion
    productAttributeAssign(productTypeId:"${productTypeId}", operations:{
```
It was okay without the comma? Alternatively the formatting is off

**Review by Cloud11PL — CHANGES_REQUESTED** [review]

**Review by Cloud11PL — APPROVED** [review]

File: cypress/integration/configuration/productTypes.js [hunk 3]
```diff
@@ -4,24 +4,38 @@
...
+import { getDefaultChannel } from "../../support/api/utils/channelsUtils";
+import {
+  createProductInChannel,
+  deleteProductsStartsWith
+} from "../../support/api/utils/products/productsUtils";
 import filterTests from "../../support/filterTests";
 import { createProductType } from "../../support/pages/productTypePage";
 
 filterTests({ definedTags: ["all"] }, () => {
   describe("Tests for product types", () => {
```
Comment (eCzerniak) [inline cypress/integration/configuration/productTypes.js:27]:
```suggestion
  describe("As an admin I want to manage product types", () => {
```

**Review by eCzerniak — CHANGES_REQUESTED** [review]
Please change the naming of test cases as it should be. 
e.g As an admin I should be able to create product type without shipping required

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**